simconnect-sdk/src/domain/client_event.rs

```diff
@@ -182,13 +286,39 @@ impl From<ClientEvent> for (ClientEventRequest, i32) {
             ClientEvent::Brakes => (ClientEventRequest::Brakes, 0),
             ClientEvent::BrakesLeft => (ClientEventRequest::BrakesLeft, 0),
             ClientEvent::BrakesRight => (ClientEventRequest::BrakesRight, 0),
-            ClientEvent::AxisLeftBrakeSet { value } => {
-                (ClientEventRequest::AxisLeftBrakeSet, value)
-            }
-            ClientEvent::AxisRightBrakeSet { value } => {
-                (ClientEventRequest::AxisRightBrakeSet, value)
-            }
+            ClientEvent::AxisLeftBrakeSet { value } => (ClientEventRequest::AxisLeftBrakeSet, value),
+            ClientEvent::AxisRightBrakeSet { value } => (ClientEventRequest::AxisRightBrakeSet, value),
             ClientEvent::ParkingBrakes => (ClientEventRequest::ParkingBrakes, 0),
+            // Comm Radios
+            ClientEvent::Com1RadioStbySet { value } => (ClientEventRequest::Com1RadioStbySet, value),
+            ClientEvent::Com2RadioStbySet { value } => (ClientEventRequest::Com2RadioStbySet, value),
+            ClientEvent::Com3RadioStbySet { value } => (ClientEventRequest::Com3RadioStbySet, value),
+            ClientEvent::Com1RadioSwap => (ClientEventRequest::Com1RadioSwap, 0),
+            ClientEvent::Com2RadioSwap => (ClientEventRequest::Com2RadioSwap, 0),
+            ClientEvent::Com3RadioSwap => (ClientEventRequest::Com3RadioSwap, 0),
+            // Nav Radios
+            ClientEvent::Nav1RadioStbySet { value } => (ClientEventRequest::Nav1RadioStbySet, value),
+            ClientEvent::Nav2RadioStbySet { value } => (ClientEventRequest::Nav2RadioStbySet, value),
+            ClientEvent::Nav3RadioStbySet { value } => (ClientEventRequest::Nav3RadioStbySet, value),
+            ClientEvent::Nav4RadioStbySet { value } => (ClientEventRequest::Nav4RadioStbySet, value),
+            ClientEvent::Nav1RadioSwap => (ClientEventRequest::Nav1RadioSwap, 0),
+            ClientEvent::Nav2RadioSwap => (ClientEventRequest::Nav2RadioSwap, 0),
+            ClientEvent::Nav3RadioSwap => (ClientEventRequest::Nav3RadioSwap, 0),
+            ClientEvent::Nav4RadioSwap => (ClientEventRequest::Nav4RadioSwap, 0),
+            // Transponder
+            ClientEvent::TransponderSet { value } => (ClientEventRequest::TransponderSet, value)
         }
     }
 }
+
+/// SimConnect Event Flags for TransmitClientEvent
+/// 
+/// https://docs.flightsimulator.com/html/Programming_Tools/SimConnect/API_Reference/Events_And_Data/SimConnect_TransmitClientEvent.htm
```

Comment:
This is standard functionality from PREPAR3D, so I guess it makes sense to link to the original documentation https://www.prepar3d.com/SDKv5/sdk/simconnect_api/references/general_functions.html#SimConnect_TransmitClientEvent.

Reply:
It's a little awkward, as it really dates back to MS FSX.  There's probably going to come a point where Prepar3D and MSFS diverge in the future (if it hasn't already), will be an interesting bridge to cross.